Marlin/stepper_indirection.h

```diff
@@ -472,8 +472,14 @@
   #endif
 #else
   #define E_STEP_WRITE(v) E0_STEP_WRITE(v)
-  #define NORM_E_DIR() E0_DIR_WRITE(!INVERT_E0_DIR)
-  #define REV_E_DIR() E0_DIR_WRITE(INVERT_E0_DIR)
+  #if ENABLED(MK2_MULTIPLEXER)
```

Comment:
I don't see how this will ever work. The previous pre-processor checks will ensure that `EXTRUDERS` is 1, but SanityCheck.h requires that `EXTRUDERS` be greater than 1. It seems like there should be a section immediately before or after the `SWITCHING_EXTRUDER` section in this block that defines these macros depending on the number of extruders, which all use the `E0_XXXXX` macros and invert the even-numbered steppers. Could potentially configure the multiplexer as part of `E_STEP_WRITE` here as well.